### PR TITLE
Apply ThemeManager to ChromeWindow #1843

### PR DESCRIPTION
**Pull request by tommy5oo**

### CODE REVIEW

File: nugu-ux-kit/src/main/java/com/skt/nugu/sdk/platform/android/ux/widget/EllipsizedTextView.kt [hunk 1]
```diff
@@ -31,6 +36,18 @@ class EllipsizedTextView @JvmOverloads constructor(
         setSingleLine()
     }
 
+    fun applyThemeAttrs(@StyleRes resId: Int) {
+        val attrs =
+            intArrayOf(R.attr.textColor, R.attr.textColorHint)
+        val a: TypedArray = context.obtainStyledAttributes(resId, attrs)
+        try {
+            setTextColor(a.getColor(0, Color.parseColor("#404858")))
```
Comment (Yuseon-Han) [inline nugu-ux-kit/src/main/java/com/skt/nugu/sdk/platform/android/ux/widget/EllipsizedTextView.kt:44]: default color could be static. parsing everytime is not necessary

File: nugu-ux-kit/src/main/java/com/skt/nugu/sdk/platform/android/ux/widget/ChromeWindowContentLayout.kt [hunk 2]
```diff
@@ -209,4 +210,42 @@ class ChromeWindowContentLayout @JvmOverloads constructor(
...
+     * Sets the dark mode.
+     */
+    fun setDarkMode(newDarkMode: Boolean) {
+        applyThemeAttrs(
+            when (newDarkMode) {
+                true -> R.style.Nugu_Widget_Chrome_Window_Dark
+                false -> R.style.Nugu_Widget_Chrome_Window_Light
+            }
+        )
+        sttTextView.applyThemeAttrs(
```
Comment (Yuseon-Han) [inline nugu-ux-kit/src/main/java/com/skt/nugu/sdk/platform/android/ux/widget/ChromeWindowContentLayout.kt:238]: 원본 텍스트
ChromeWindowContentLayout doesn't have to know appropriate resource of sttTextView and ChipsView.
I think It is better that only sttTextView and ChipsView know it's own style resource. ChromeWindowContentLayout better just let them know theme.

File: nugu-ux-kit/src/main/java/com/skt/nugu/sdk/platform/android/ux/widget/EllipsizedTextView.kt (hunk 1 above)
Comment (tommy5oo) [reply nugu-ux-kit/src/main/java/com/skt/nugu/sdk/platform/android/ux/widget/EllipsizedTextView.kt]: Right! thanks. (updated)

File: nugu-ux-kit/src/main/java/com/skt/nugu/sdk/platform/android/ux/widget/ChromeWindowContentLayout.kt (hunk 2 above)
Comment (tommy5oo) [reply nugu-ux-kit/src/main/java/com/skt/nugu/sdk/platform/android/ux/widget/ChromeWindowContentLayout.kt]: Ok apply your review. 
Thanks

**Review by Yuseon-Han — APPROVED** [review]

**Pull request merged**